packages/compass-connections/src/connections-manager.ts

```diff
@@ -157,6 +157,12 @@ export class ConnectionsManager extends EventEmitter {
     );
   }
 
+  getConnectionIdsByStatus(status: ConnectionStatus): ConnectionInfoId[] {
+    return [...this.connectionStatuses.entries()]
+      .filter(([, connectionStatus]) => connectionStatus === status)
+      .map(([connectionId]) => connectionId);
+  }
```

Comment:
This adds one more way to access this information besides our hooks - `useActiveConnections`, `useConnectionsWithStatus`. I think I understand the usecase here but we would want to be very deliberate about this because it might be fine but otherwise a tech debt (either this or the hooks?) that needs to be cleaned up. We don't need to block the PR on this but it would be a good idea to populate the compass-connections ticket with this information as well.

Reply:
I see your point, but the use cases are quite different - this is a simple one-time getter, similar to `statusOf`, while the hooks are maintaining a state and listening to updates

Reply:
Totally might be missing some code in this PR, but is it being used by something? I don't mind that we're adding this method here, but as we're going to change the manager too in the follow-up refactors, if we don't use it, I'd maybe skip adding more methods to it